USERS.md

```diff
@@ -24,125 +24,126 @@ Currently, the following organizations are **officially** using Argo CD:
-1. [New Relic](https://newrelic.com/)
-1. [Nextdoor](https://nextdoor.com/)
-1. [Nikkei](https://www.nikkei.co.jp/nikkeiinfo/en/)
-1. [Octadesk](https://octadesk.com)
-1. [openEuler](https://openeuler.org)
-1. [openGauss](https://opengauss.org/)
-1. [openLooKeng](https://openlookeng.io)
-1. [OpenSaaS Studio](https://opensaas.studio)
-1. [Opensurvey](https://www.opensurvey.co.kr/)
-1. [Optoro](https://www.optoro.com/)
-1. [Orbital Insight](https://orbitalinsight.com/)
-1. [PayPay](https://paypay.ne.jp/)
-1. [Peloton Interactive](https://www.onepeloton.com/)
-1. [Pipefy](https://www.pipefy.com/)
-1. [Polarpoint.io](https://polarpoint.io)
-1. [Preferred Networks](https://preferred.jp/en/)
-1. [Prudential](https://prudential.com.sg)
-1. [PUBG](https://www.pubg.com)
-1. [Qonto](https://qonto.com)
-1. [QuintoAndar](https://quintoandar.com.br)
-1. [Quipper](https://www.quipper.com/)
-1. [Recreation.gov](https://www.recreation.gov/)
-1. [Red Hat](https://www.redhat.com/)
-1. [Riskified](https://www.riskified.com/)
-1. [Robotinfra](https://www.robotinfra.com)
-1. [Saildrone](https://www.saildrone.com/)
-1. [Saloodo! GmbH](https://www.saloodo.com)
-1. [Schwarz IT](https://jobs.schwarz/it-mission)
-1. [Speee](https://speee.jp/)
-1. [Spendesk](https://spendesk.com/)
-1. [Sumo Logic](https://sumologic.com/)
-1. [Swisscom](https://www.swisscom.ch)
-1. [Swissquote](https://github.com/swissquote)
-1. [Syncier](https://syncier.com/)
-1. [TableCheck](https://tablecheck.com/)
-1. [Tailor Brands](https://www.tailorbrands.com)
-1. [Tesla](https://tesla.com/)
-1. [ThousandEyes](https://www.thousandeyes.com/)
-1. [Ticketmaster](https://ticketmaster.com)
-1. [Tiger Analytics](https://www.tigeranalytics.com/)
-1. [Toss](https://toss.im/en)
-1. [tru.ID](https://tru.id)
-1. [Twilio SendGrid](https://sendgrid.com)
-1. [tZERO](https://www.tzero.com/)
-1. [UBIO](https://ub.io/)
-1. [UFirstGroup](https://www.ufirstgroup.com/en/)
-1. [Universidad Mesoamericana](https://www.umes.edu.gt/)
-1. [Viaduct](https://www.viaduct.ai/)
-1. [Virtuo](https://www.govirtuo.com/)
-1. [VISITS Technologies](https://visits.world/en)
-1. [Volvo Cars](https://www.volvocars.com/)
-1. [VSHN - The DevOps Company](https://vshn.ch/)
-1. [Walkbase](https://www.walkbase.com/)
-1. [WeMo Scooter](https://www.wemoscooter.com/)
-1. [Webstores](https://www.webstores.nl)
-1. [Whitehat Berlin](https://whitehat.berlin) by Guido Maria Serra +Fenaroli
-1. [Witick](https://witick.io/)
-1. [WSpot](https://www.wspot.com.br/)
-1. [Yieldlab](https://www.yieldlab.de/)
-1. [Zimpler](https://www.zimpler.com/)
-1. [Sap Labs](http://sap.com)
-1. [Smilee.io](https://smilee.io)
-1. [Metanet](http://www.metanet.co.kr/en/)
-1. [Unifonic Inc](https://www.unifonic.com/)
-1. [Tamkeen Technologies](https://tamkeentech.sa/)
-1. [Kaltura](https://corp.kaltura.com/)
-1. [Boticario](https://www.boticario.com.br/)
-1. [Beleza Na Web](https://www.belezanaweb.com.br/)
-1. [MariaDB](https://mariadb.com)
+2. [Chime](https://chime.com)
```

Comment:
Numbered lists in markdown do not require the sequence to increase, this makes it easier to insert/remove items. Can you revert that change?

```
1. Foo
1. Bar
1. Baz
```

is rendered as
1. Foo
1. Bar
1. Baz

Reply:
Closing this PR and creating a new one